backend/molgenis-emx2-io/src/main/java/org/molgenis/emx2/io/MolgenisIO.java

```diff
@@ -108,4 +102,20 @@ public static Task fromClasspathDirectory(
       String path, Schema schema, boolean strict, String... includeTableNames) {
     return fromStore(new TableStoreForCsvFilesClasspath(path), schema, strict, includeTableNames);
   }
+
+  public static class OutputAllStrategy {
+
+    private boolean includeSystemColumns = false;
```

Comment:
the use of name Strategy suggested to me that you would have two implementations of OutputAll(conform the strategy pattern). However, this is not the case. I think this is not a Strategy but more like a Options right?